### ISSUE TITLE
[Suggestions] Speed Increase & Block Configuration 

### ISSUE THREAD

**Issue by Squirtle8459** [Thutmose/Thut#34]
As addressed by the title, there should be a speed increase.. However it should function differently than you would probably think I mean.. For example, if I have 3 floors and two floors (1 & 2) are both about 12 blocks from each other in elevation than the elevator can move at a fixed speed of whatever it is normally and if I go from another floor set (1 & 3) the distance could be about 90 blocks and the speed would scale in addition to the added block distance so like (fixed_rate * (Math.abs(prev_block_y - new_block_y)/100))

basically if the fixed rate is like movement speed is a float, hopefully.. it would be something like 5f as a fixed rate.. and the distance being 90 blocks away would change the speed to 5.9f..

As for the other suggestions, 'block configuration' is in reference to the controller, basically if possible it just changes the colors/shape of the 'buttons' so they are squares or triangles.. and instead of black text on white background, it could be red text on black background or some other options, merely for aesthetic reasons.
